### PR TITLE
2015 07 mn add alexanderplatz

### PR DESCRIPTION
**Pull request by MagdaN**
This adds some documents needed for alexanderplatz

### CODE REVIEW

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/MeinBerlin.ts [hunk 1]
```diff
@@ -7,10 +8,12 @@ export var moduleName = "adhMeinBerlin";
...
     AdhMeinBerlinKiezkassen.register(angular);
     AdhMeinBerlinBplaene.register(angular);
+    AdhMeinBerlinAlexanderplatz.register(angular);
 
     angular
         .module(moduleName, [
             AdhMeinBerlinKiezkassen.moduleName,
-            AdhMeinBerlinBplaene.moduleName
+            AdhMeinBerlinBplaene.moduleName,
+            AdhMeinBerlinAlexanderplatz.moduleName
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/MeinBerlin.ts]: I would prefer if these were alphabetically ordered.

**Comment by xi** [comment]
I can not say much about the badge json. The rest looks good.

**Comment by MagdaN** [comment]
changed the order

**Comment by xi** [comment]
I guess as the badges are not final anyway this can just as well be merged.

**Pull request merged**